daml-assistant/integration-tests/src/DA/Daml/Assistant/CreateDamlAppTests.hs

```diff
@@ -95,9 +95,13 @@ tests =
           step "Install dependencies for UI"
           createDirectoryIfMissing True "node_modules"
           cachedDeps <- locateRunfiles (mainWorkspace </> "daml-assistant" </> "integration-tests" </> "create_daml_app_deps.tar")
-          runConduitRes
-              $ sourceFileBS cachedDeps
-              .| Tar.Conduit.Extra.untar (Tar.Conduit.Extra.restoreFile (\a b -> fail (T.unpack $ a <> " " <> b)) ".")
+          -- This crashes when updating NodeJS and I don't have time or
```

Comment:
Maybe just remove it then? Doing it in haskell vs calling out to `tar` isn't better or more "noble" anyway.